values/gatekeeper-operator/constraint-templates/k8sallowedrepos.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
```

Comment:
all gotmpl files should have `.gotmpl` extension

Reply:
files removed